job/server/src/main/java/alluxio/master/job/JobMaster.java

```diff
@@ -257,15 +257,17 @@ public synchronized void run(JobConfig jobConfig, long jobId)
   }
 
   /**
-   * Cancels a job.
+   * Cancels a job. Does nothing if the job doesn't exist.
```

Comment:
Don't we throw `JobDoesNotExistException` still?

Reply:
Oh oops. It used to in the middle. And then, I went back so let me remove this.